docs/user-guide/configuration/jobconfiguration.md

```diff
@@ -75,6 +75,8 @@ jobs:
             - teardown-mystep2: echo world
 ```
 
+## Teardown
```

Comment:
can you also add a menu item like https://github.com/screwdriver-cd/guide/pull/431/files#diff-f91f2320924ddba4b2acdba311b562cfb520acf90874430ec772c6aca790121aR29-R31

Reply:
menu item for teardown? its inside job-configuration, which is also a sub-menu item. Do we need to add there?

Reply:
yes. see how it's done elsewhere.